server/models/account.js

```diff
@@ -48,12 +63,12 @@ exports.update = (
 ) =>
   db.none(
     `UPDATE account
-      SET first_name    = $1,
-          last_name     = $2,
-          year          = $3,
-          picture       = $4,
-          concentration = $5,
-          pronouns      = $6
+      SET first_name    = COALESCE($1, first_name),
+          last_name     = COALESCE($2, last_name),
+          year          = COALESCE($3, year),
+          picture       = COALESCE($4, picture),
+          concentration = COALESCE($5, concentration),
+          pronouns      = COALESCE($6, pronouns)
```

Comment:
This probably isn't a concern for us, but I just wanted to _put_ it out there that this kind of handling is good for a PATCH endpoint, but I think for PUT we usually want to provide the entire updated resource and error out if it's an in invalid resource.

See: https://rapidapi.com/blog/put-vs-patch/

Reply:
Looks good, this could be really useful for the settings page.

Reply:
Ah, true